### ISSUE TITLE
Use better PID tracking errors on PSM for last 4 joints

### ISSUE THREAD

**Issue by adeguet1** [jhu-dvrk/sawIntuitiveResearchKit#26]
The current implementation has a very large PID tracking error for the last 4 joints.  This is mostly because the current implementation for the engage procedure of the tool and adapter don't use any trajectory generation (see also issue #2 ).  One temporary fix is to add an extra step after the engage procedure to let some time after the direct PID command to finish before turning the PID tracking tolerance to a more reasonable number (10 degrees for example).

**Comment by adeguet1** [comment]
Partial fix in 8571756209bba957f58d45d85589700d70a756fd. (in devel branch).

This is only a partial fix because the tracking errors I had to use were quite large.  I noticed that the PID errors were large in normal operations and I think the PID gains could be significantly increased.  I'm reluctant to increase the PID gains right now since we had some hardware failures on this joints.  We should probably lower the max torque on the IO first (feature to be implemented) and then increase the PID gains.

On the plus side, even with the wimpy gains we have on the last 4 joints, the robot is quite stiff so we know we can reduce the max torque on IO.  In abnormal situations, i.e. very large PID error, the gains we have can still lead to a maximum torque on the last 4 motors.
